gpflow/models/svgp.py

```diff
@@ -132,26 +131,27 @@ def prior_kl(self):
                                           self.q_sqrt,
                                           whiten=self.whiten)
 
-    def log_likelihood(self, X: tf.Tensor, Y: tf.Tensor) -> tf.Tensor:
+    def log_likelihood(self, data: Tuple[tf.Tensor, tf.Tensor]) -> tf.Tensor:
         """
         This gives a variational bound on the model likelihood.
         """
+        x, y = data
```

Comment:
```suggestion
        X, Y = data
```
I'm in favour of using the upper-case variables to indicate that these are matrices